nucypher/cli/commands/stake.py

```diff
@@ -106,6 +107,7 @@
 option_lock_periods = click.option('--lock-periods', help="Duration of stake in periods.", type=click.INT)
 option_worker_address = click.option('--worker-address', help="Address to bond as an Ursula-Worker", type=EIP55_CHECKSUM_ADDRESS)
 option_index = click.option('--index', help="The staker-specific stake index to edit", type=click.INT)
+option_only_lock = click.option('--only-lock', help="Locking tokens without deposit", is_flag=True)
```

Comment:
I find "only lock" a bit awkward.  How about "Deposit Only" (This is a common phrase for this purpose in the US at least)?

Reply:
"only lock" is more "without deposit", not sure how to name it